user/views.py

```diff
@@ -63,8 +68,8 @@ def post(self, request):
             user            = User.objects.get(email=email)
             user_password   = user.password.encode('utf-8')
 
-            if not user.is_allowed:
-                return JsonResponse({'message': 'UNAUTHORIZED_USER'}, status=401)
+            # if not user.is_allowed:
+            #     return JsonResponse({'message': 'UNAUTHORIZED_USER'}, status=401)
```

Comment:
주석이 추가된 이유는 뭔가요?

Reply:
이부분만 확인되면 merge 할 수 있을 것 같아요!

Reply:
회원가입 후 이메일 인증을 통해 해당 계정을 활성화시키는 기능을 구현하였는데 프로젝트 진행상 힘들 것 같습니다
일단 지금은 구현할 수 없는 기능이라 주석으로 처리해놨습니다

Reply:
이런 경우는 나중에 할 일로,  `# TODO` : ...로 표시해주시면 좋아요~

```suggestion
        # TODO : ...로 표시해주시면 좋아요~
            # if not user.is_allowed:
            #     return JsonResponse({'message': 'UNAUTHORIZED_USER'}, status=401)
```

Reply:
고생 많으셨습니다~